web/src/models/TestRun.model.ts

```diff
@@ -58,14 +58,28 @@ const getTestResultCount = (
 
 export function isRunStateFinished(state: TTestRunState) {
   return (
-    [TestState.FINISHED, TestState.TRIGGER_FAILED, TestState.TRACE_FAILED, TestState.ASSERTION_FAILED] as string[]
```

Comment:
The BE has this function https://github.com/kubeshop/tracetest/blob/main/server/model/run.go#L119 which is used for every finish test run status